code/control/DeployPlanDispatcher.php

```diff
@@ -290,6 +284,12 @@ protected function getGitPrevDeploys($project) {
 	 * @return SS_HTTPResponse
 	 */
 	protected function getAPIResponse($output, $statusCode) {
+		$secToken = $this->getSecurityToken();
+		$secToken->reset();
```

Comment:
that is an excellent point that I should have thought off. :headdesk: will find a better solution.